Documentation/app-container.md

```diff
@@ -0,0 +1,23 @@
+## App Container basics
+
+[App Container][appc-repo] is a [specification][appc-spec] of an image format, runtime, and discovery protocol for running applications in containers.
+
+Rocket implements the two runtime components of the specification: the [Application Container Executor (ACE)][appc-ace] and the [Metadata Service][app-meta].
+
+It also leverages schema and code from the upstream [appc/spec][appc-spec] repo to manipulate ACIs, work with manifests, and perform image discovery.
+
+## Validating Rocket
+
+To validate that `rkt` successfully implements the ACE part of the spec, use the App Container [validation ACIs][appc-readme]:
```

Comment:
Possibly "Validation of `rkt`'s conformity to the specification can be done using the commands from the App Container [validation ACIs][appc-readme]:"

More just a nitpick.  Starting the phrase with a preposition made me totally misread it the first time and I had to go back and re-read it.